translations/prompt.txt/cs.po

```diff
@@ -18,20 +18,12 @@ msgstr ""
 
 #. type: Plain text
 #: tools/translation-ai-prompts/prompt.txt:4
-#, fuzzy
-#| msgid ""
-#| "We're translating board game rule book from English to [...].  The text "
-#| "is in LaTeX format, and the messages are in GNU gettext po format.  "
-#| "Output just the translated `msgstr` given the `msgid`, maintaining the "
-#| "original formatting and quotes.  Rules:"
 msgid ""
 "We're translating board game rule book from English to [...].  The text is "
 "in LaTeX format, and the messages are in GNU gettext po format.  Rules:"
 msgstr ""
-"We're translating board game rule book from English to Czech.  The text is "
-"in LaTeX format, and the messages are in GNU gettext po format.  Output just "
-"the translated `msgstr` given the `msgid`, maintaining the original "
-"formatting and quotes.  Rules:"
+"Překládáme pravidla deskové hry z Angličtiny do Češtiny. Text je ve formátu "
```

Comment:
This part is actually not necessary to translate. Since all the rest of the prompt is in English, in fact, maybe it's better to leave it?